tpm2/tpm2_test.go

```diff
@@ -1363,3 +1363,112 @@ func TestCreatePrimaryRawTemplate(t *testing.T) {
 		t.Errorf("got key exponent %v, want %v", pubRSA.E, defaultKeyParams.RSAParameters.Exponent)
 	}
 }
+
+func TestMatchesTemplate(t *testing.T) {
+	var tests = []struct {
```

Comment:
`tests := []struct {`

only use var at package level or for declaring zero values